x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/bulk_actions/use_bulk_actions.tsx

```diff
@@ -137,7 +137,13 @@ export const useBulkActions = ({
           type: BulkActionType.duplicate,
           duplicatePayload: {
             include_exceptions:
-              modalDuplicationConfirmationResult === DuplicateOptions.withExceptions,
+              modalDuplicationConfirmationResult === DuplicateOptions.withExceptions ||
+              modalDuplicationConfirmationResult ===
+                DuplicateOptions.withExceptionsExcludeExpiredExceptions,
+            include_expired_exceptions: !(
+              modalDuplicationConfirmationResult ===
+              DuplicateOptions.withExceptionsExcludeExpiredExceptions
+            ),
```

Comment:
nit: Maybe have `useBulkDuplicateExceptionsConfirmation`'s `showBulkDuplicateConfirmation` return object also include `include_exceptions`/`include_expired_exceptions` bools thus encapsulating this logic?

Was just thinking how to DRY this out since it's used a few places, no biggie either way.